whipper/test/test_common_config.py

```diff
@@ -21,23 +21,25 @@ def tearDown(self):
 
     def testAddReadOffset(self):
         self.assertRaises(KeyError,
-            self._config.getReadOffset, 'PLEXTOR ', 'DVDR   PX-L890SA', '1.05')
+                          self._config.getReadOffset, 'PLEXTOR ',
```

Comment:
I would move `'PLEXTOR ',` to the next line and maybe move `self._config.…` to the line above (if that still stays within the limit) or just keep that on its own line. It's nicer on the (well, my :)) eyes to have the drive info together.